packages/react-components/react-toast/src/components/Timer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { makeStyles } from '@griffel/react';
+
+const useStyles = makeStyles({
+  progress: {
+    animationName: {
+      from: {
+        opacity: 0,
+      },
+      to: {
+        opacity: 0,
+      },
+    },
+  },
+});
+
+export const Timer: React.FC<{
+  running: boolean;
+  timeout: number;
+  onTimeout: () => void;
+}> = props => {
```

Comment:
`TimerProps`?!

Reply:
This is only a temporary component, and even in the final implementation it will only be internal